socorro/processor/processor_app.py

```diff
@@ -156,7 +156,7 @@ class ProcessorApp(FetchTransformSaveApp):
         reference_value_from='secrets.sentry',
     )
 
-    def _capture_error(self, crash_id, exc_info):
+    def _capture_error(self, exc_info, crash_id=None):
```

Comment:
I switched the order of these so I could make `crash_id` optional. When processing crashes, it makes sense to have the `crash_id` as context--that makes errors in processing soooo much more helpful. But for errors in the queue code, it's not a thing.